checkstyle/suppressions.xml

```diff
@@ -39,9 +39,12 @@
     <!-- Cache -->
     <suppress checks="MethodCount" files="com/hazelcast/config/CacheConfig"/>
     <suppress checks="MethodCount" files="com/hazelcast/cache/ICache"/>
+    <suppress checks="MethodCount" files="com/hazelcast/cache/impl/ICacheRecordStore"/>
```

Comment:
These changes need to be added to the class itself. We are trying to move away from the supressions.xml